composer.json

```diff
@@ -7,15 +7,15 @@
   "prefer-stable": true,
   "require": {
     "php": "^8.1.0",
-    "symfony/console": "6.2.*"
+    "symfony/console": "^5.4",
+    "symfony/http-kernel": "^5.4",
+    "symfony/config": "^5.4",
+    "symfony/dependency-injection": "^5.4"
```

Comment:
@ismail1432 how can you register this bundle without require these dependencies in `require` section?